src/main/java/com/silverlining/orderservice/service/OrderDetailsServiceImpl.java

```diff
@@ -1,32 +1,33 @@
 package com.silverlining.orderservice.service;
 
 import com.silverlining.orderservice.constants.OrderStatus;
-import com.silverlining.orderservice.dto.Cart;
-import com.silverlining.orderservice.dto.OrderDetailDto;
-import com.silverlining.orderservice.dto.OrderDto;
-import com.silverlining.orderservice.dto.ProductDto;
+import com.silverlining.orderservice.dto.*;
 import com.silverlining.orderservice.models.Order;
 import com.silverlining.orderservice.models.OrderDetail;
 import com.silverlining.orderservice.repository.OrderDetailsRepository;
+import com.silverlining.orderservice.utils.OrderUtilities;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.modelmapper.ModelMapper;
 import org.modelmapper.convention.MatchingStrategies;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.ResponseEntity;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.*;
 import org.springframework.stereotype.Component;
 import org.springframework.transaction.annotation.Transactional;
 import org.springframework.web.client.RestTemplate;
 
 import java.time.LocalDateTime;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
 import java.util.stream.Collectors;
 
 @Component
 public class OrderDetailsServiceImpl implements OrderDetailsService{
 
     private static final Logger LOGGER = LogManager.getLogger(OrderDetailsServiceImpl.class);
+    public static final String PRODUCTS_ALL_URL = "http://product-ws/products/all";
```

Comment:
move these constants to another class called as Contant.java

Reply:
done